src/Uno.UI/UI/Xaml/Controls/ToolTip/ToolTipService.cs

```diff
@@ -167,6 +167,13 @@ private static void OnOwnerLoaded(object sender, RoutedEventArgs e)
 			{
 				owner.PointerEntered += OnPointerEntered;
 				owner.PointerExited += OnPointerExited;
+				owner.Tapped += OnTapped;
```

Comment:
It is been noted that only some controls dismiss the tooltip on press, namely the clickable ones from `ButtonBase` and some other controls such as `TextBox` or `ComboBox`. Others like `ListView` or `Slider` kept the tooltip opened through clicks.

And, it is not necessarily the Tapped/KeyDown events that turn off the tooltip. For instance, `TextBox` only hides tooltip on TextChanged. Arrows keys to (which would trigger KeyDown) doesnt cause the tooltip to be hidden.

Reply:
I checked again in  Windows App.

- As you say, only `ButtonBase` close tooltip when pointer pressed. 
- And other controls including `ListView` and `Slider` close tooltip on click.
- Keyboard events (except arrows) always close Tooltip.
- There are no other cases.

@Xiaoy312 , I want your advice before editing the code.

Reply:
- `ListView`: To expand on this, hovering an LVItem to have the tooltip being shown, and then clicking the same LVI does not close the tooltip, but clicking a different one will do.
- `Slider`: Tooltip are kept opened while dragging **until released**, if you start dragging from the hover spot that is away from the thumb.
- keyboard: There are some other exceptions in addition to arrow keys, like: <kbd>alt</kbd><kbd>ins/del/home/end</kbd>
  But, only if they don't change the focus. So, keeping KeyDown would be better choice, than other specific events, as it is more precise and applicable to most controls (from UIElement).

Reply:
![lv slider tooltip](https://github.com/unoplatform/uno/assets/2359550/fa36180a-86c6-4c51-bac6-e1b37a38437a)

Reply:
It is fine if you can't perfectly replicate the expected behaviors for LV/LVItem or Slider. You can open issue(s) about the discrepancies that remains.